.gitignore

```diff
@@ -17,8 +17,8 @@ default.etcd
 job_data_example/config/
 .vscode/
 .idea/
-internal/pkg/protofiles/client_CP/*.pb.go
-internal/pkg/protofiles/executor_CP/*.pb.go
+internal/pkg/protofiles/client_CP/*.go
```

Comment:
pb.go needs to be pushed, @faruqisan suggested to keep the pb.go files to avoid compile time errors

Reply:
yes please.